microhapulator/pipeaux.py

```diff
@@ -264,13 +264,13 @@ def aggregate_summary(samples, reads_are_paired=True):
             frmaptotal == maptotal
         ), f"Sample={sample} fullrefr map total={frmaptotal} map total={maptotal}"
         typing = pd.read_csv(f"analysis/{sample}/{sample}-typing-rate.tsv", sep="\t")
-        num_typed_reads = typing.TypedReads.sum()
-        typing_total_reads = typing.TotalReads.sum()
+        num_typing_success = typing.TypedReads.sum()
+        num_typing_attempted = typing.TotalReads.sum()
```

Comment:
Just changed the variable names here to be more consistent with the new table columns.